frontends/web/src/locales/en/app.json

```diff
@@ -24,7 +24,17 @@
     "extendedPublicKey": "Extended public key",
     "label": "Account info",
     "title": "Account information",
-    "verify": "Verify on device"
+    "verify": "Verify on device",
+    "xpubTypeChangeBtn": {
+      "p2pkh": "View legacy P2PKH extended public key",
+      "p2wpkh": "Switch back to Native segwit (bech32)",
```

Comment:
maybe also `View` instead of `Switch back`? then the order does not matter.

Reply:
I prefer `Switch back` as it goes back to the default one, but looking forward when we add another scripttype you're right it's better to make them all the same.